clusters/build-clusters/02_cluster/installation_params.md

```diff
@@ -0,0 +1,60 @@
+# Build02: installation parameters
+
+## [Configuring a GCP project](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html)
```

Comment:
We want `build02` on GCP because:
- We e2e tests for GCP. It might be convenience to run the tests there too.
- `build01` is on AWS. It might be a good timing to see how good/back it works on GCP.

Reply:
yup, I think doing it on GCP is a very good idea